packages/mdc-toolbar/mdc-toolbar.scss

```diff
@@ -21,7 +21,7 @@
 
 $mdc-toolbar-height: 64px;
```

Comment:
Should we rename these two variables to `$mdc-toolbar-row-height` and `$mdc-toolbar-mobile-row-height` respectively?